### DISCUSSION TITLE
How to become an official reseller

### DISCUSSION THREAD

**Discussion by lazaj** [BlueSCSI/BlueSCSI-v2#100] [General]
Hello,

Let's me introduce myself, I am senior engineer in electronic and embedded software.
Actually I work as a freelance, I also so teach routing (in altium) and IPC.
I have a huge collection of vintage Macs, Unix workstations, Amiga, ...
On my spare time, I repair vintage computers for collectors who don't have electronics skills.
Over time, I have provided bluescsi to many collectors. I make my own bluescsi. I don't do this to get rich but to preserve the old computers that we love.
I build my bluescsi in batches of 50 pieces, I often have excess pieces.
I would like to become an "official" seller, I live in France. Not to make money but to participate in your project.

**Comment by erichelgeson** [comment]
Hi @lazaj, glad you like the project and are using the spirit of the non-commercial license to help your collection and others. Currently we are not looking for new sellers in the EU. This question does get asked from time to time so I'll add a page on bluescsi.com with more info.
